vmm/src/config.rs

```diff
@@ -727,6 +729,15 @@ impl CpusConfig {
                     features.amx = true;
                     Ok(())
                 }
+                #[cfg(target_arch = "aarch64")]
+                "sve" => {
+                    if is_aarch64_feature_detected!("sve") || is_aarch64_feature_detected!("sve2") {
```

Comment:
Rather than checking at config parsing time (which will ignore if the option is given via the HTTP JSON API) it  should be done at VM boot time (in the configure logic.)